src/java.instrument/share/classes/sun/instrument/InstrumentationImpl.java

```diff
@@ -36,7 +36,6 @@
 import java.nio.file.Path;
 import java.nio.file.InvalidPathException;
 import java.net.URL;
-import java.security.AccessController;
 import java.security.CodeSource;
 import java.security.PrivilegedAction;
```

Comment:
A note for future cleanups in this area: This `java.security.PrivilegedAction` import should probably have been removed.